Tools/CISupport/kill-old-processes

```diff
@@ -165,14 +165,16 @@ def main(user=None):
     # This allows to specify on the environment of the bot a list of process that should be killed.
     tasksToKill = os.getenv("WEBKITBOT_TASKSTOKILL", "").split()
     if sys.platform == 'darwin':
-        os.system("ps aux")
-        for task in tasksToKill + tasksToKillMac:
-            os.system("killall -9 -v -m " + task)
         # Shutdown any simulators
         if os.geteuid() == 0 and user:
             os.system("sudo -u {} xcrun simctl shutdown booted".format(user))
```

Comment:
This only shuts down one simulator, not all of them. Also, I think that this just starts the shutdown, and we need to wait for it to finish.

Reply:
Will swap `booted` for `all`.

I'll put together a function that checks if any simulators are booted, but we'll need a timeout, we still want to kill all other processes if simulators are failing to shut down.